DigitalLearningSolutions.Data.Tests/Services/GroupServiceTests/GroupsServiceTests.cs

```diff
@@ -522,12 +789,102 @@ private void SetUpAddCourseEnrolProcessFakes(GroupCourse groupCourse)
                     A<int?>._
                 )
             ).Returns(groupCourse.GroupCustomisationId);
-            
+
             A.CallTo(() => groupsDataService.GetGroupDelegates(A<int>._))
                 .Returns(new List<GroupDelegate> { reusableGroupDelegate });
 
-            A.CallTo(() => groupsDataService.GetGroupCourseIfVisibleToCentre(groupCourse.GroupCustomisationId, centreId))
+            A.CallTo(
+                    () => groupsDataService.GetGroupCourseIfVisibleToCentre(groupCourse.GroupCustomisationId, centreId)
+                )
                 .Returns(groupCourse);
         }
+
+        private void SetUpGenerateGroupFakes(
+            DateTime timeNow,
+            IEnumerable<CustomPrompt>? customPrompts = null,
+            IEnumerable<Group>? groupsAtCentre = null,
+            int newGroupId = 1

```

Comment:
I don't think this parameter is ever specified anywhere, so the default is always used. Might as well get rid of it and use the new const instead.

Reply:
done